roles/test_collectd/tasks/main.yml

```diff
@@ -28,3 +28,11 @@
   register: stat
   changed_when: false
   failed_when: stat.stdout_lines|length == 0
+
+- name: TEST Check collectd memory limit
+  ansible.builtin.shell: |
+    {{ container_bin }} inspect {{ collectd_container_name }} | grep -i \"Memory\"
+  register: memory
+  changed_when: false
+  failed_when: memory.stdout_lines != 536870912
```

Comment:
I'm not 100% sure how exactly failed_when works, but if you grep on `Memory` then one line of the stdout will be `Memory: 536870912` , correct? I wonder then how this can work :). Does the `!=` works as `!contains`?